brave/brave-core
# Refactor Brave Ads ConfirmationsObserver to ConfirmationsDelegate

Resolves https://github.com/brave/brave-browser/issues/19946

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Confirm ads attempts to top up unblinded tokens after successfully sending a confirmation to the server
- Confirm ads attempts to top up unblinded tokens after failing to send a confirmation to the server
